manager/README.md

```diff
@@ -25,8 +25,378 @@ The service is configured using the environment variables from the following tab
 | MANAGER_INSTANCE_ID      | Manager service instance ID                              |                                   |
 | COCOS_MESSAGE_BROKER_URL | message broker url                                       | nats://localhost:4222             |
 
+## Setup
+
+```sh
+git clone https://github.com/ultravioletrs/cocos
+cd cocos
+```
+
+NB: all relative paths in this document are relative to `cocos` repository directory.
+
+### QEMU-KVM
+
+[QEMU-KVM](https://www.qemu.org/) is a virtualization platform that allows you to run multiple operating systems on the same physical machine. It is a combination of two technologies: QEMU and KVM.
+
+- QEMU is an emulator that can run a variety of operating systems, including Linux, Windows, and macOS.
+- [KVM](https://wiki.qemu.org/Features/KVM) is a Linux kernel module that allows QEMU to run virtual machines.
+
+To install QEMU-KVM on a Debian based machine, run
+
+```sh
+sudo apt update
+sudo apt install qemu-kvm
+```
+
+Create `img` directory in `cmd/manager`. Create `tmp` directory in `cmd/manager`.
+
+### Prepare focal-server-cloudimg-amd64.img
+
+*focal-server-cloudimg-amd64* is a `qcow2` file with Ubuntu server preinstalled, ready to use with the QEMU virtual machine. We will put [Agent](https://github.com/ultravioletrs/cocos/agent) in the *focal-server-cloudimg-amd64*. In order to do that, we need to prepare the disk image.
+
+#### Download focal-server-cloudimg-amd64.img and set up the root password
+
+First, we will download *focal-server-cloudimg-amd64*. It is a `qcow2` file with Ubuntu server preinstalled, ready to use with the QEMU virtual machine.
+
+```sh
+cd cmd/manager
+# Use this dir for focal-server-cloudimg-amd64.img and a test copy of firmware vars file.
+mkdir img
+# Use this dir for temporary firmware vars file and temporary disk image per virtual machine.
+mkdir tmp
+
+FOCAL=focal-server-cloudimg-amd64.img
+wget -O img/$FOCAL https://cloud-images.ubuntu.com/focal/current/$FOCAL
+# focal-server-cloudimg-amd64 comes without the root password
+sudo apt-get install libguestfs-tools
+PASSWORD=coolpass
+sudo virt-customize -a img/$FOCAL --root-password password:$PASSWORD
+```
+#### Resize disk image, partition and filesystem
```

Comment:
Can you please explain why we need to resize the partition and filesystem? Do we need to do it in order to have enough space for golang and agent.

Reply:
Yes we need some additional space for agent golang dependencies before we build agent